pkg/sql/plan/types.go

```diff
@@ -77,6 +77,20 @@ type BaseOptimizer struct {
 	ctx   CompilerContext
 }
 
+type ExecType int
+
+const (
+	ExecTypeAP ExecType = iota
+	ExecTypeTP
+)
+
+type ExecInfo struct {
+	Typ        ExecType
+	WithGPU    bool
```

Comment:
I was just using an example.   I don't think we want it here,  if really needed we will add it later.   I think a suggest of number of CNs are indeed useful now.

Reply:
Haha, gpu indeed may never be involved put.